mmpose/models/detectors/bottom_up.py

```diff
@@ -236,3 +243,117 @@ def forward_test(self, img, img_metas, **kwargs):
         image_path.extend(img_metas['image_file'])
 
         return results, scores, image_path
+
+    def show_result(self,
+                    img,
+                    result,
+                    skeleton=None,
+                    kpt_score_thr=0.3,
+                    pose_kpt_color=None,
```

Comment:
since `None` is not a valid value for `pose_kpt_color` and `pose_limb_color` in this function, it is better to either
* do not provide defaults in the signature
* assert that they are not None
* provide default colors when None

Reply:
I prefer option 3 btw

Reply:
change to "draw kpt&limbs, only if the color is not None".
And add `assert` to the color.